### ISSUE TITLE
Dockerfile doesn't build any more

### ISSUE THREAD

**Issue by rlangfordBV** [sysdiglabs/kube-psp-advisor#23]
**Describe the bug**
The `Dockerfile` doesn't build successfully

**To Reproduce**
1. `docker build -t kube-advisor -f ./container/Dockerfile .`

**Expected behavior**
A built docker image

**What actually happens**
```
Step 9/12 : RUN env GOOS=$(uname -s | tr '[:upper:]' '[:lower:]') GOARCH=amd64 CGO_ENABLED=0 go build -a
 ---> Running in 817c5f18704c
# k8s.io/client-go/transport
../k8s.io/client-go/transport/round_trippers.go:70:11: cannot convert klog.V(9) (type klog.Verbose) to type bool
../k8s.io/client-go/transport/round_trippers.go:72:11: cannot convert klog.V(8) (type klog.Verbose) to type bool
../k8s.io/client-go/transport/round_trippers.go:74:11: cannot convert klog.V(7) (type klog.Verbose) to type bool
../k8s.io/client-go/transport/round_trippers.go:76:11: cannot convert klog.V(6) (type klog.Verbose) to type bool
The command '/bin/sh -c env GOOS=$(uname -s | tr '[:upper:]' '[:lower:]') GOARCH=amd64 CGO_ENABLED=0 go build -a' returned a non-zero code: 2
```

**Comment by Kaizhe** [comment]
@rlangfordBV thanks for reporting this. It is a known issue: https://github.com/kubernetes/client-go/issues/654 

I will fix it.

**Comment by Kaizhe** [comment]
fix in #25
